### ISSUE TITLE
Feature Request: Option to display caption information

### ISSUE THREAD

**Issue by wesf99** [mvysny/photocloud-frame-slideshow#157]
Is there an option to show the IPTC Description/Caption information while playing the slideshow. If not, could it be added please.

**Comment by mvysny** [comment]
Unfortunately PhotoCloud only support the EXIF metadata at the moment. However, there is a Java library for reading and writing IPTC metadata, so perhaps it would be possible to add support for this: https://stackoverflow.com/questions/3851665/java-library-for-reading-and-writing-iptc-metadata-to-jpeg-and-tiff

Can you please attach a photo which contains the IPTC information to this ticket, so that I have a proper image for testing purposes?

**Comment by wesf99** [comment]
Sounds promising. See attached photo and Screen print of a windows program, ExifPro 2.1, showing the same data. Wes

**Comment by mvysny** [comment]
Sorry, the photo doesn't seem to be attached. Probably you need to attach it via the GitHub's web interface.

**Comment by wesf99** [comment]
I'll try again. 
![IPTC Comment Caption or Description Data Example](https://user-images.githubusercontent.com/55278725/118460360-337b5a00-b72f-11eb-8399-898c627210d5.PNG)
![ConniesPhotos005](https://user-images.githubusercontent.com/55278725/118460408-40984900-b72f-11eb-9191-5d43d38e3b56.jpg)
![IPTC Comment Caption or Description Data Example](https://user-images.githubusercontent.com/55278725/118460475-54dc4600-b72f-11eb-9dfb-e11e9fe6adcc.PNG)
![ConniesPhotos005](https://user-images.githubusercontent.com/55278725/118460588-6cb3ca00-b72f-11eb-827c-57cccba5c706.jpg)
Sounds promising. See attached photo and Screen print of a windows program, ExifPro 2.1, showing the same data. Wes

**Comment by wesf99** [comment]
Opps I inadvertently closed the issue

**Comment by wesf99** [comment]
Did you get the photo and screen shot through github? Wes

**Comment by mvysny** [comment]
Yes, thank you. I was able to read the IPTC data on my computer as follows:
```
118460588-6cb3ca00-b72f-11eb-827c-57cccba5c706.jpg:
 Tag      Name                 Type      Size  Value
 -------- -------------------- --------- ----  -----
 1:090    Coded Character Set  Binary       3  1b 25 47
 2:000    Record Version       Short        2  4
 2:005    Object Name          String      27  Grandmother Lunn nee Powell
 2:055    Date Created         Date         8  18900402
 2:060    Time Created         Time         6  123025
Mother of Ethel Fielding,  Grandmother of Olive, Reg, Jack, Kenneth, Desmond and Connie Fieiding
```
I'll try to implement a support for IPTC into PhotoCloud :+1:

**Comment by wesf99** [comment]
I was just wondering if you were able to get the caption feauture working?

**Comment by mvysny** [comment]
Sorry I had no time to work on this, but hopefully this week
